### PR TITLE
added react

### PR DESCRIPTION
**Pull request by Arjav-Patel**

### CODE REVIEW

File: frontend/public/manifest.json [hunk 1]
```diff
@@ -0,0 +1,25 @@
+{
+  "short_name": "React App",
+  "name": "Create React App Sample",
+  "icons": [
+    {
```
Comment (SwappyG) [inline frontend/public/manifest.json]: should this have a better name?

**Review by SwappyG — COMMENTED** [review]
My Javascript knowledge is too weak to comment much, but i'll ask sanity questions
- should all the html be hardcoded the way it is?
- what is the `package-lock.json`, why is it `14,092` lines, and should it be version controlled?

File: frontend/public/manifest.json (hunk 1 above)
Comment (Arjav-Patel) [reply frontend/public/manifest.json]: Most of these files are automatically generated (like package-lock and this one), so I didn't bother changing them. Changed the name to dnd tho.

**Comment by Arjav-Patel** [comment]
I'm not sure on what the optimal way to use react with bootstrap is, since I've been using react documentation to create this app, which is not the ideal way to learn a new tool.

What do you mean hardcoded though?

**Comment by SwappyG** [comment]
> I'm not sure on what the optimal way to use react with bootstrap is, since I've been using react documentation to create this app, which is not the ideal way to learn a new tool.
> 
> What do you mean hardcoded though?

I just saw a bunch of HTML looking code and magic numbers right inside functions, and was wondering if that made sense to do. But it's alright, just go ahead and merge this

**Review by SwappyG — APPROVED** [review]

**Comment by SwappyG** [comment]
Oh, i guess about the package_lock, make sure it's not something that's always autogenerated. if it is, then probably should add it to `.gitignore`

**Comment by SwappyG** [comment]
The project has probably changed too much since we semi-abandoned it a year ago. Closing this PR, but will use some of the ideas here for the revamp.

**Pull request closed without merging**